plugins.json

```diff
@@ -1,10 +1,10 @@
 {
   "images/dominant-color-images": {
     "slug": "dominant-color-images",
-    "version": "1.0.0"
+    "version": "1.0.1"
   },
   "images/webp-uploads": {
     "slug": "webp-uploads",
-    "version": "1.0.4"
+    "version": "1.0.5"
   }
 }
```

Comment:
Can you please also update the readme for these two and add a changelog entry pointing to this PR?

Reply:
Missed it 😞. Thanks for pointing. Added changes on readme in https://github.com/WordPress/performance/pull/866/commits/a77d3e97dfa64175fc83efd8377bc221b4e3dddb